tests/monotouch-test/monotouch-test.csproj

```diff
@@ -310,6 +311,8 @@
     <BundleResource Include="xamvideotest.mp4" />
     <BundleResource Include="CoreImage\xamarinmonkey.heic" />
     <BundleResource Include="Resources\Base.lproj\Localizable.strings" />
+    <BundleResource Include="uncompressed.js" />
+    <BundleResource Include="compressed_lze" />
```

Comment:
Does the uncompressed file really have to be this big? Can't you just include a small file of random text instead?

Also it seems the `uncompressed.js` file contains copyrighted text:

>  "copyright": "Copyright &copy; 2017 Apple Inc. All rights reserved.",

Reply:
Good point, I'll fix this.